### DISCUSSION TITLE
Where can I find model-viewer-effects?

### DISCUSSION THREAD

**Discussion by arperevalov** [google/model-viewer#4262] [Q&A]
Found page at [official website](https://modelviewer.dev/examples/postprocessing/), but I can't find npm package of this enhancement. 

https://github.com/google/model-viewer/tree/master/packages/model-viewer-effects shows that npm is invalid. Is it released yet? If no, when it will be?

**Comment by elalish** [comment]
Nope, it's pre-release: https://twitter.com/modelviewer/status/1651007361862225921 Expect it in a month or two - we need more people testing it first. Note iOS just shipped a bug that broke it too.
